build.sbt

```diff
@@ -1,18 +1,21 @@
-import com.typesafe.config._
 
+import com.typesafe.sbt.packager.MappingsHelper.directory
+
+
+mappings in Universal ++= directory(baseDirectory.value / "public")
 name := "delphi-webapp"
 
 version := "1.0.0-SNAPSHOT"
```

Comment:
When you do a release you should set version number.

Reply:
Version number set to 1.1.0